isis/src/hayabusa2/apps/hyb2onccal/Hyb2OncCalUtils.h

```diff
@@ -221,9 +221,14 @@ namespace Isis {
     }
 
     double smear = 0;
-    for (int j = 0; j < imageIn.size(); j++) {
+    int skipCount = 0;
+    for (int i = 0; i < imageIn.size(); i++) {
+      if (IsSpecial(imageIn[i])) {
```

Comment:
Suggest pulling the divide out of the loop and putting below, so all of the smear is divided by the same number.